### PR TITLE
refactor(HeaderMessages): update to functional component

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add one more header data json to test the alerts.

### CODE REVIEW

File: src/components/Header/HeaderMessages/HeaderMessages.js [hunk 1]
```diff
@@ -1,62 +1,44 @@
...
-}
+            {alert.button.text}
+          </button>
+        ) : (
+          <></>
+        )}
+      </div>
+      <Modal className="modal" isOpen={buyModalIsOpen} handleClose={() => toggleModal(false)}>
+        <UpgradePlanForm
+          isSubscriber={user.plan.isSubscribers}
```
Comment (cbernat) [inline src/components/Header/HeaderMessages/HeaderMessages.js]: Did yoy test that you always have a plan? how about free users?

**Comment by cbernat** [comment]
👍

File: src/components/Header/HeaderMessages/HeaderMessages.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Header/HeaderMessages/HeaderMessages.js]: yes they have a plan

**Pull request merged**